release_notes/ocp_3_5_release_notes.adoc

```diff
@@ -2400,3 +2400,71 @@ route, and the data accessed is limited to the indices.
 To upgrade an existing {product-title} 3.4 or 3.5 cluster to this latest
 release, use the automated upgrade playbook. See
 xref:../install_config/upgrading/automated_upgrades.adoc#running-the-upgrade-playbook-directly[Performing Automated In-place Cluster Upgrades] for instructions.
+
+[[ocp-3-5-rhba-2017-3438]]
+=== RHBA-2017:3438 - Moderate: {product-title} 3.5.5.31.48 Bug Fix and Enhancement Update
+
+Issued: 2017-12-13
+
+{product-title} release 3.5.5.31.48 is now available. The list of packages
+included in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2017:3438[RHBA-2017:3438] advisory.
+The container images included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2017:3439[RHBA-2017:3439] advisory.
+
+Space precluded documenting all of the bug fixes, enhancements, and images for
+this release in the advisories. See the following sections for notes on
+upgrading and details on the bug fixes and images included in this release.
+
+[[ocp-3-5-rhba-2017-3438-images]]
+==== Images
+
+This release updates the Red Hat Container Registry
+(`registry.access.redhat.com`) with the following images:
+
+----
+openshift3/logging-curator:v3.5.5.31.48-2
+openshift3/metrics-deployer:v3.5.5.31.48-3
+openshift3/node:v3.5.5.31.48-2
+openshift3/openvswitch:v3.5.5.31.48-2
+openshift3/ose-base:v3.5.5.31.48-2
+openshift3/ose-deployer:v3.5.5.31.48-2
+openshift3/ose-docker-builder:v3.5.5.31.48-2
+openshift3/ose-docker-registry:v3.5.5.31.48-2
+openshift3/ose-egress-router:v3.5.5.31.48-2
+openshift3/ose-f5-router:v3.5.5.31.48-2
+openshift3/ose-haproxy-router:v3.5.5.31.48-2
+openshift3/ose-keepalived-ipfailover:v3.5.5.31.48-2
+openshift3/ose-pod:v3.5.5.31.48-2
+openshift3/ose-sti-builder:v3.5.5.31.48-2
+openshift3/ose:v3.5.5.31.48-2
+----
+
+[[ocp-3-5-rhba-2017-3438-bug-fixes]]
+==== Bug Fixes
+
+* Custom openshift-ansible modules were not being loaded due to a missing dependency configuration. The issue was first identified when using Ansible 2.4.1, which included a change in how tasks and roles were dynamically included at runtime.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1516469[*1516469*])
```

Comment:
The BZ formatting is consistent in this section as '******' but it was previously written 'BZ#******'. I'm not sure if that is super important though.